src/obj.rs

```diff
@@ -146,7 +150,9 @@ where
         let mut materials = HashMap::new();
 
         for m in &self.material_libs {
-            let file = match File::open(&m) {
+            let mut full_path = self.path.clone();
+            full_path.push(&m);
+            let file = match File::open(&full_path) {
```

Comment:
could just do `self.path.join(&m)` if the path is for the folder

Reply:
:+1: